docs/json_to_sarif.md

```diff
@@ -0,0 +1,151 @@
+| `$.warnings[*].check_name`          | `TBD`                           | "DefaultRoutes"                                                                        |
+| `$.warnings[*].message`             | `TBD`                           | "Any public method in `FooPutController` can be used as an action for `put` requests." |
+| `$.warnings[*].file`                | `TBD`                           | "config/routes.rb"                                                                     |
+| `$.warnings[*].line`                | `TBD`                           | null, 1                                                                                |
+| `$.warnings[*].link`                | `TBD`                           | "https://brakemanscanner.org/docs/warning_types/default_routes/"                       |
+| `$.warnings[*].code`                | `TBD`                           | "User.find(params[:id]).bio", "params[:bad_stuff]", null                               |
+| `$.warnings[*].render_path`         | `TBD`                           | `TODO`                                                                                 |
+| `$.warnings[*].location`            | `TBD`                           | `TODO`                                                                                 |
+| `$.warnings[*].user_input`          | `TBD`                           | "params[:user_input]", null                                                            |
+| `$.warnings[*].confidence`          | `TBD`                           | "Medium"                                                                               |
+| `$.ignored_warnings[*]`             | `TBD`                           | `TODO`                                                                                 |
+| `$.errors[*]`                       | `TBD`                           | `TODO`                                                                                 |
+| `$.obsolete[*]`                     | `TBD`                           | `TODO`                                                                                 |
+
+## Questions
+
+- Does Brakeman have a sense of severity that we can map into the results? Should we default to "warning" for all results?
+- What does `$.warnings[*].render_path` represent? Should we map this onto an appropriate SARIF property (there may be a better place, but this _could_ land in a _property bag_ under results, i.e. `$.runs[*].results[*].properties`)?
+- We need the absolute physical location of a file containing a result. It looks like this is available from `$.warnings[*].file` and `$.warnings[*].line`, but as a relative location. How can we derive the absolute location from this?
+- The `$.warnings[*].location` doesn't always seem to correspond directly with `$.warnings[*].file` / `$.warnings[*].location`, what is the reason for this redirection? Should we include both locations in the SARIF results?
```

Comment:
`location` is not very reliable. It's meant to be the class/module/template where the warning is located. I would recommend not including it. File+line is best.